source/predicates/predicate.ts

```diff
@@ -4,6 +4,8 @@ import {not} from '../operators/not';
 import {BasePredicate, testSymbol} from './base-predicate';
 import {Main} from '..';
 
+export type ValidatorMessage<T> = (value: T, label?: string) => string;
```

Comment:
Needs a doc comment.

Reply:
@GentileFulvio I still believe you should fix this. I don't think it should be marked as `hidden` because it's a type that is exposed and can be used by users of `ow`.

Reply:
Added some docs, not sure if it's good enough though